File: src/apps/Identity/Nexus.Account.Web/Scripts/features/register-new-user/model/register-user.request.ts
export interface RegisterRequest {
    Login: string,
    UserName: string,
    Verifier: string,
    ClientSalt: string,
    EncryptedDek: string,
    EncryptionAlgorithm: string,
    Iterations: number,
    KdfType: string,
    Email: string,
    Phone: string,
    IdGender: string,
    IdCountry: string
}

File: contracts/src-dotnet/Rebout.Nexus.Contracts/UserManagement/v1/RegisterUserRequest.cs
namespace Rebout.Nexus.Contracts.UserManagement.v1
{
    public record RegisterUserRequest(string Login, string UserName, string Verifier, string ClientSalt, string EncryptedDek, string EncryptionAlgorithm, int Iterations, string KdfType, string Email, string? Phone, string? IdGender, string? IdCountry);
}